### PR TITLE
Update README.me

### PR DESCRIPTION
**Pull request by DisenoDeError**
First task in the mentorship program, changes made to development branch in README file

### CODE REVIEW

**Review by borisbradic — CHANGES_REQUESTED** [review]
You must restrict merge on 'main' branch, current set up allows merging without approval.

**Comment by DisenoDeError** [comment]
> You must restrict merge on 'main' branch, current set up allows merging without approval.

changes made to need approval before merging to main

**Review by borisbradic — APPROVED** [review]
Now's all good.

**Pull request merged**
